src/frame/content.rs

```diff
@@ -58,6 +60,14 @@ impl Content {
         }
     }
 
+    /// Returns the `ExtendedLink` or None if the value is not `ExtendedLink`.
```

Comment:
Paste oops in comment, could you please `s/ExtendedLink/EncapsulatedObject/`?

Reply:
How about adding the line from the spec to the code as a comment? This reasoning was not immediately obvious to us apparently. Since it applies to both PartialEq and Hash were could it be placed?

Reply:
Yes, good idea. Although I think it makes more sense to link to the spec in the type definition